docs/mesos/README.md

```diff
@@ -7,56 +7,64 @@
 <!--- end of master only -->
 
 # Mesos with Calico Networking
-**Calico provides IP-Per-Container networking for your Mesos cluster.** The following collection of tutorials will walk you through the steps necessary to get up and running.
+**Calico provides IP-Per-Container networking for Mesos clusters.** The following collection of tutorials will walk through the steps necessary for installation and use.
 
-Note: IP-Per-Container Networking with Calico is an opt-in feature for Mesos Frameworks that launch tasks with [Networkinfo](https://github.com/apache/mesos/blob/0.26.0-rc3/include/mesos/mesos.proto#L1383). This means that your favorite Mesos Frameworks will not work with Calico until they have opted to include Networkinfo when launching tasks. Marathon is the first Framework planned to use this new feature, but its Mesos-networking support is not yet complete. 
+Questions? Contact us on the #Mesos channel of [Calico's Slack](https://calicousers-slackin.herokuapp.com/).
 
-Calico support is under development. Use the following information to ensure you choose the right version:
-- Calico fully supports and recommends Mesos 0.26
-- Calico supports Mesos 0.25, but we recommend against using it as there aren't any Frameworks (including Marathon) which support the Networkinfo specs from 0.25 (which were modified for 0.26)
-- Calico works on Mesos 0.24, but only as a proof of concept, and is no longer supported.
+### Mesos Version Compatability
+Calico support is actively being developed. Use the following information to ensure you choose the right version:
+- **Mesos 0.26:** Recommended. Full Calico Support for the future.
+- **Mesos 0.25:** Deprecated. Calico works with Mesos 0.25, but we recommend against using it as there aren't any Frameworks (including Marathon) which support the Networkinfo specs from 0.25 (which were modified for 0.26)
+- **Mesos 0.24:** Unsupported. Calico works as a proof of concept, but is no longer supported.
 
-Have any questions on these tutorials? Is there any material you want to see covered? Let us know on our [Slack channel](https://calicousers-slackin.herokuapp.com/).
+### Support for Adding Calico to an Existing Mesos Cluster
+The following tutorials cover installing Netmodules on a fresh machine which does not yet have Mesos installed.
+This is because Calico requires Netmodules, a Mesos Networking Module, which must be compiled with the source header files of the active Mesos installation in order to ensure compatability. If your source installation files exist on your Mesos Slave, then it is possible to follow steps 1-3 and step 6 of the [Manual Calico-Mesos installation Guide](ManualInstallCalicoMesos.md) to add Netmodules and Calico to your existing cluster.
```

Comment:
While we're here, can you change this to: "At this time, we do not support adding netmodules to an existing mesos cluster."

We're gong to be working on it next week. But it was sort of a lie writing this in the first place.